.github/workflows/test.yaml

```diff
@@ -0,0 +1,14 @@
+name: "Test"
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - uses: actions/setup-python@v2
```

Comment:
this might be confusing on why we are setting python for non-python projects

Reply:
This template is not language specific.